packages/projection/README.md

```diff
@@ -1,20 +1,100 @@
+
+### ObservableCardanoNode
+
+This is our interface to Chain Sync and Local State Query protocols of Cardano node. The only implementation available in `cardano-js-sdk` is `OgmiosObservableCardanoNode`.
+
+## Utilities
+
+### Bootstrap.fromCardanoNode
+
+Creates a projection source (`Observable<ProjectionEvent>`) from:
+- [ObservableCardanoNode](#observablecardanonode)
+- [StabilityWindowBuffer](#stabilitywindowbuffer)
+- `Observable<TipOrOrigin>` that emits block header of last processed event (local tip). This is used to find intersection point.
+
+```mermaid
+---
+title: Bootstrap Algorithm
+---
+flowchart LR
+    subscribe[Subscribe] --> find-intersect{Find node\nintersection\nwith local tip}
+    find-intersect --> |Found| start[Start ChainSync\nfrom intersection]
+    start --> get-event{Get event\nfrom node,\ncheck type}
+    get-event --> |RollBackward| get-block[Get block from StabilityWindowBuffer]
+    get-event --> |RollForward| hydrate
+    start --> hydrate[Hydrate event\nwith additional\npropertes]
+    hydrate --> emit[Emit\nProjectionEvent]
+    find-intersect --> |Not Found| get-block[Get block from\nStabilityWindowBuffer]
+    get-block --> rollback[Create\nRollBackward\nevent]
+    rollback --> hydrate
+    rollback --> wait[Wait for\nlocal tip change]
+    wait --> is-rollback-point{Is rollback point?}
+    is-rollback-point --> |No| get-block
+    is-rollback-point --> |Yes| started-sync{Already started\nChain Sync?}
+    started-sync --> |Yes| get-event
+    started-sync --> |No| find-intersect
+```
+
+### Mappers
+
+Projections are typically 
+1. Interpreting the block (e.g. extracting some relevant properties from all transactions). `Mappers` is a namespace that contains a collection of RxJS operators that process the ProjectionEvent (block) and emit a new event object that has some additional properties.
+2. Doing something with it (e.g. store into the database). 
+
+Here's an example:
+
+```typescript
+// source$: Observable<ProjectionEvent<{}>>
+source$.pipe(
+  // policyId is a parameter that should be provided
+  // by handle issuer, e.g. ADA Handle
+  Mappers.withHandles({ policyId }),
+  // type WithHandles = { handles: HandleOwnership; }
+  // Event type can also be inferred (no need for explicit type declaration)
+  tap((event: ProjectionEvent<WithHandles>) => {
+    console.log('Handles found by withHandles operator', event.handles);
+  })
+)
 ```
+
+### Other Utils
+
+#### InMemoryStabilityWindowBuffer
+
+If your application doesn't need persistence, you can use in-memory implementation of [StabilityWindowBuffer](#stabilitywindowbuffer), which can also be used as local tip tracker for Bootstrap (provides `Observable<TipOrOrigin>`).
+
+--- 
+*<sup>1</sup>Currently our projection source observable is not a pure 'Producer', as events have a `requestNext` method that is used to control the source. We'll probably be changing how this works (LW-7812).*
```

Comment:
Is LW-7812 accessible to the public? If not, we might want to say something like:
Maybe we can say:
> This is subject to change in the future.

Reply:
Done in https://github.com/input-output-hk/cardano-js-sdk/pull/952/commits/e8d9e3ac77b682ce213d9fc817c72f56893acf14